tests/__snapshots__/i18n.test.js.snap

```diff
@@ -109,18 +83,44 @@ exports[`Routing creates Netlify redirects 1`] = `
 /getStaticProps/withFallbackBlocking/3  /.netlify/functions/next_getStaticProps_withFallbackBlocking_id  200!  Cookie=__prerender_bypass,__next_preview_data
 /getStaticProps/withFallbackBlocking/3  /en/getStaticProps/withFallbackBlocking/3  200
 /getStaticProps/withFallbackBlocking/4  /.netlify/functions/next_getStaticProps_withFallbackBlocking_id  200!  Cookie=__prerender_bypass,__next_preview_data
 /getStaticProps/withFallbackBlocking/4  /en/getStaticProps/withFallbackBlocking/4  200
-/getStaticProps/withFallbackBlocking/:id  /.netlify/functions/next_getStaticProps_withFallbackBlocking_id  200
 /getStaticProps/withRevalidate/1  /.netlify/functions/next_getStaticProps_withRevalidate_id  200
 /getStaticProps/withRevalidate/2  /.netlify/functions/next_getStaticProps_withRevalidate_id  200
+/static  /en/static.html  200
+/_next/image*  url=:url w=:width q=:quality  /.netlify/functions/next_image?url=:url&w=:width&q=:quality  200
+/api/shows/:id  /.netlify/functions/next_api_shows_id  200
+/api/shows/:params/*  /.netlify/functions/next_api_shows_params  200
+/en/getServerSideProps/all  /.netlify/functions/next_getServerSideProps_all_slug  200
+/en/getServerSideProps/all/*  /.netlify/functions/next_getServerSideProps_all_slug  200
+/en/getServerSideProps/:id  /.netlify/functions/next_getServerSideProps_id  200
+/en/getStaticProps/withFallback/:id  /.netlify/functions/next_getStaticProps_withFallback_id  200
+/en/getStaticProps/withFallback/:slug/*  /.netlify/functions/next_getStaticProps_withFallback_slug  200
+/en/getStaticProps/withFallbackBlocking/:id  /.netlify/functions/next_getStaticProps_withFallbackBlocking_id  200
+/en/getStaticProps/withRevalidate/withFallback/:id  /.netlify/functions/next_getStaticProps_withRevalidate_withFallback_id  200
+/en/shows/:id  /.netlify/functions/next_shows_id  200
+/en/shows/:params/*  /.netlify/functions/next_shows_params  200
+/en/static/:id  /en/static/[id].html  200
+/es/getServerSideProps/all  /.netlify/functions/next_getServerSideProps_all_slug  200
+/es/getServerSideProps/all/*  /.netlify/functions/next_getServerSideProps_all_slug  200
+/es/getServerSideProps/:id  /.netlify/functions/next_getServerSideProps_id  200
+/es/getStaticProps/withFallback/:id  /.netlify/functions/next_getStaticProps_withFallback_id  200
+/es/getStaticProps/withFallback/:slug/*  /.netlify/functions/next_getStaticProps_withFallback_slug  200
+/es/getStaticProps/withFallbackBlocking/:id  /.netlify/functions/next_getStaticProps_withFallbackBlocking_id  200
+/es/getStaticProps/withRevalidate/withFallback/:id  /.netlify/functions/next_getStaticProps_withRevalidate_withFallback_id  200
+/es/shows/:id  /.netlify/functions/next_shows_id  200
+/es/shows/:params/*  /.netlify/functions/next_shows_params  200
+/es/static/:id  /es/static/[id].html  200
+/getServerSideProps/all  /.netlify/functions/next_getServerSideProps_all_slug  200
+/getServerSideProps/all/*  /.netlify/functions/next_getServerSideProps_all_slug  200
```

Comment:
Note to myself: This static route is here, because the `getNetlifyRoutes` function generates two redirects for pages with optional catch-all routing: One for the "naked" route (no params) and one for the catch-all option. It's not a bug and it's not a problem 😊